test/e2e/http-error-message.test.js

```diff
@@ -4,7 +4,7 @@ const VlHttpErrorMessagePage = require('./pages/vl-http-error-message.page');
 describe('vl-http-error-message', async () => {
     const vlHttpErrorMessagePage = new VlHttpErrorMessagePage(driver);
 
-    before(() => {
+    beforeEach(() => {
```

Comment:
Wat loopt er fout als we dit niet doen?

Reply:
Een ingevulde waarde in het formulier wordt niet leeggemaakt (omdat we dat niet doen).

Reply:
Op welk formulier heb je het dan? Want dit is toch de http-error-message.

Reply:
oei... Ik verwarde dit met een vl-form-validation (omdat ik daar ook zoiets deed). Bij de laatste test werd de pagina op 't einde opnieuw geladen (omdat er iets ervoor via een click naar een andere pagina werd gegaan).

Reply:
Ahja, klopt